aea/helpers/dialogue/base.py

```diff
@@ -656,6 +688,31 @@ def new_self_initiated_dialogue_reference(self) -> Tuple[str, str]:
         """
         return str(self._dialogue_nonce + 1), ""
 
+    def create(self, counterparty, performative, **kwargs,) -> Tuple[Message, Dialogue]:
+        initial_message = self._message_class(
+            dialogue_reference=("", ""),
+            message_id=1,
+            target=0,
```

Comment:
should be expressed in constants (we already have it i think)

Reply:
Done